### DISCUSSION TITLE
ExportToCellbrowser in Seurat 4

### DISCUSSION THREAD

**Discussion by Rodcarlessi** [satijalab/seurat#4864] [Q&A]
Dear Seurat Team,

I am trying to export datasets created in Seurat 4 to Cellbrowser. I note ExportToCellbrowser moved to SeuratWrappers in Seurat 4, however even calling the function from SeuratWrappers I keep getting the following error:

Object was created with Seurat version 4.0.0
can only process Seurat2 or Seurat3 objects, object was made with Seurat 4.0.0

Any suggested solution for this?

Thank you,
Rodrigo

**Comment by millersan** [comment]
Use seurat 2 or 3 and UpdateSeuratObject() instead.
